the-name-card/src/components/EditProfile/EditProfile.tsx

```diff
@@ -0,0 +1,309 @@
+import { ChangeEvent, useRef, useState } from 'react';
```

Comment:
might want to put in this component in another directory (maybe called `pages`?) to distinguish components from pages

Reply:
nit: also this file is a bit long (~300 lines) if you want to follow the good practice of keeping components leaner (100-200 lines) u might want to refactor out some parts

Reply:
on both points, will be doing a major refactoring for soon.

for the pages directory will be pulling the profile page & landing page out and placing their components in the aptly named component directory

as for the EditProfile component, the idea was to get the whole component working to solely focus on functionality before splitting to different components. As the component is quite logic heavy, I thought that it would be easier to work in a single file at first.

tldr: yes will be refactoring for maintainability and clarity